src/components/SuperTokenListItem.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import {
+  Flex,
+  Box,
+  Icon,
+  Button,
+  Link,
+  Stack,
+} from '@chakra-ui/react';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import { FaCopy } from 'react-icons/fa';
+import { RiQuestionLine } from 'react-icons/ri';
+import { VscLinkExternal } from 'react-icons/vsc';
+
+import { ToolTipWrapper } from '../staticElements/wrappers';
+import { SF_LABEL } from '../utils/toolTipLabels';
+
+const SuperTokenListItem = ({
+  token,
+  tokenAddress,
+  handleCopyToast,
+  daoMember,
+  withdrawSupertoken,
+  loading,
+  successToast,
+}) => {
+  const handleWithdraw = () => {
+    withdrawSupertoken(tokenAddress);
+  };
+
+  const displayComingSoon = () => {
+    successToast({ title: 'Coming Soon' });
+  };
+  const isLoading = loading?.active && loading?.condition === tokenAddress;
+  const shouldDisableWithdraw = token?.tokenBalance <= 0 || (loading?.active && loading?.condition === tokenAddress);
```

Comment:
another small layout thing. I think we're going with simple statements above functions.

https://daohaus.club/docs/devs/react-layount